### PR TITLE
Math Magicians: Testing

### PR DESCRIPTION
**Pull request by maov19**
## 🆕 Added features:

✔️ Set up React Testing Library
✔️ Wrote unit tests for the files operate.js and calculate.js using Jest.
✔️ Created unit tests for all React components:
✔️ Used React Testing Library to simulate user interaction.

🔧 Please consider that the following issues were fixed:

✅  The linters are all passing 💯 
✅  All tests are passing! 📷

image
 
✅ Updated README (both authors included)
 
image

### CODE REVIEW

**Review by Zuhaib042 — COMMENTED** [review]
Hi @maov19 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/tests/components.test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, fireEvent, waitForElementToBeRemoved } from '@testing-library/react';
+
+/* eslint-disable */
+import Home from '../components/home';
```
Comment (christianbiring1) [inline src/tests/components.test.js]: - [ ] Kindly make sure all your test cases are passing. as you may notice the `Home` wasn't imported correctly. please make sure you do do.
image

```suggestion
import Home from '../components/Home'; 
```

**Review by christianbiring1 — COMMENTED** [review]
Hi @maov19 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request

You have submitted a project, that was supposed to be done as a pair programming but it was done solo, Kindly make sure all the partners have contributed to the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/tests/components.test.js (hunk 1 above)
Comment (maov19) [reply src/tests/components.test.js]: thanks! its fixed now 🔧

**Comment by maov19** [comment]
## Dear reviewer,

Please consider that the following issues were fixed:
- @ZewdieMc commited on the repo 🥇 
- The linters are all passing 💯 
- All tests are passing! 📷

image
 
- Updated README (both authors included)
 
image

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @maov19 and @ZewdieMc,

✔️ User interaction tests are well-written and passing

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**